netlify/functions/save-discussion-background/Slack.ts

```diff
@@ -128,7 +128,6 @@ export namespace Slack {
         text: message,
         as_user: true,
         thread_ts: threadTS,
-        icon_url: AVATAR_URL,
```

Comment:
As per the documentation of Slack API, icon_url is accepted as an argument inside the method - https://api.slack.com/methods/chat.postMessage